src/stdlib/src/stdlib.hpp

```diff
@@ -25,4 +25,22 @@ namespace perlang
     void print(const BigInt& bigint);
     void print(float f);
     void print(double d);
+
+    BigInt BigInt_pow(const BigInt& value, int exponent)
+    {
+        // "With constant auxiliary memory" implementation as given in
+        // https://en.wikipedia.org/wiki/Exponentiation_by_squaring
```

Comment:
This is actually not true, we currently have the plain recursive implementation. I'll fix this now to use the slightly better version described on that page.

Reply:
Fixed and amended into the original commit, resolving.